src/website/app/demos/Grid/bestPractices.js

```diff
@@ -36,16 +37,16 @@ components to a columnar layout.`,
       description: `Don't use Grid if the content being laid out should _not_
 [respond to language direction](#rtl).`,
       example: (
-        <Grid css={{ height: 200 }}>
+        <Grid className={css({ height: 200 })}>
```

Comment:
Same as Flex's best practices; please change to use `height` prop (sorry).

Reply:
Fixed, no problem...